src/test/java/test/TestBase.java

```diff
@@ -0,0 +1,136 @@
+package test;
+
+import client.ApiClientException;
+import client.ConfirmationCodeApiClient;
+import io.appium.java_client.MobileElement;
+import io.appium.java_client.android.AndroidDriver;
+import io.appium.java_client.remote.MobileCapabilityType;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+import pages.ConfirmationCodePage;
+import pages.SetPinKePage;
+import pages.SignUpKePage;
+import pages.SplashScreenKePage;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public abstract class TestBase {
+
+    public static String KE_PREFIX = "712";
```

Comment:
Same trouble as in original repo: TestBase are overloaded with responsibilities. Just a note no need to fix